### PR TITLE
Overriding default artifact_id to help healthcheck pass

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

**Comment by cschroed-usgs** [comment]
Going to merge this without review since this may be the first of a few whack-a-mole attempts to solve this problem on our servers.

**Comment by cschroed-usgs** [comment]
@zmoore-usgs this fixed the health check for the validator on the test tier. Should I repeat this for all docker images?

File: Dockerfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ ENV listening_port=6027
 ENV protocol=https
 ENV oauth_server_token_key_url=https://example.gov/oauth/token_key
 ENV authorized_roles=test_default
+ENV artifact_id=usgs-wma-mlr-validator
```
Comment (cschroed-usgs) [inline Dockerfile]: I'm not crazy about duplicating the artifact id value, but I don't know how to get it out of setup.py once we're in the run stage of the multi-stage build. Do you have any ideas?

Comment (zmoore-usgs) [reply Dockerfile]: That seems reasonable to me. I don't know of a way to do that either.. Perhaps there's a way to get the python build process to spit out a text file with a well-known name that we can also copy over and have the run image `cat` or something? But that definitely could be more effort than it's worth vs. just doing this solution. I don't have a strong opinion either way

Comment (cschroed-usgs) [reply Dockerfile]: Allrighty, same page then. I think it's acceptable to duplicate the values for the time being. If we implement ML-474 then devs would learn pretty quickly post-deploy if the two values were out of sync.

**Pull request merged**